.coffeelintignore

```diff
@@ -1,2 +0,0 @@
-iconsets
-node_modules
```

Comment:
Let's ignore it just in case. It still searches in there for `.js` files and we never know what happens when sets get updated or added.